src/main/java/uk/gov/dvsa/model/cvs/VTP30Bilingual.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.dvsa.model.cvs;
+
+public class VTP30Bilingual extends VTP30W {
+
+    public String getTestType() {
+        return "PSV";
+    }
+
+    public String getPresentedDocumentNameFail() {
+        return "VTP30";
+    }
+
+    public String getPresentedDocumentNameFailWelsh() {
+        return "VTP30W";
+    }
+
+    public String getVersionNumberFail() {
+        return "1.0";
+    }
+
+    public String getVersionNumberFailWelsh() {
+        return "1.0";
+    }
+
+    public String getRegOrIdHeading() {
+        return "Identification number";
```

Comment:
Looking at VTP30, I believe this should be "Reference number" and not "Identification number"

Reply:
why would this be different to what's in the hgv/trl fail one?